releasenotes/releasenotes_v4.rst

```diff
@@ -3,6 +3,24 @@
 Old Firely Server release notes (v4.x)
 ======================================
 
+.. _vonk_releasenotes_4_10_1:
+
+Release 4.10.1, March 14th, 2023
+---------------------------------
+
+.. attention::
+   This is a security related release which addresses a security vulnerability in the MongoDB driver that Firely Server uses. **This update is highly recommended for all customers.**
```

Comment:
Why recommend in bold when we're not vulnerable and most customers use another database?

Reply:
My thinking was: They might switch, they might use it experimentally, they might want to compare performance, another attack vector related to this vulnerability might pop up etc. 

I updated the description and removed the boldness though. Might seem a bit too dramatic.